pkg/config/setup/config.go

```diff
@@ -110,30 +110,6 @@ var (
 	systemProbe pkgconfigmodel.Config
 )
 
-// Datadog returns the current agent configuration
-func Datadog() pkgconfigmodel.Config {
-	return datadog
-}
-
-// SetDatadog sets the the reference to the agent configuration.
-// This is currently used by the legacy converter and config mocks and should not be user anywhere else. Once the
-// legacy converter and mock have been migrated we will remove this function.
-func SetDatadog(cfg pkgconfigmodel.Config) {
-	datadog = cfg
-}
-
```

Comment:
It seems that this function is used in the `pkg/config/legacy_converter.go` file.